core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -138,9 +138,7 @@ class BlockManagerMasterEndpoint(
       }
   }
 
-  private def getTopologyInfoForHost(host: String): String = {
-    topologyMapper.getTopologyForHost(host)
-  }
+  private def getTopologyInfoForHost(h: String): String = topologyMapper.getTopologyForHost(h)
```

Comment:
I meant inline the function into the caller. It only seems to be used in one place.